Assets/TilemapHeightTest/Scripts/TileHeightManager.cs

```diff
@@ -30,7 +30,7 @@ namespace TilemapHeightTest
 {
 	public class TileHeightManager : MonoBehaviour
 	{
-		public static TileHeightManager Instance { get; set; }
+		public static List<TileHeightManager> Instances { get; set; }
```

Comment:
Could you make this `public static readonly List<TileHeightManager> Instances = new List<TileHeightManager>();`? This way, you don't have to initialize it later down in `Awake()`